docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -16,9 +16,12 @@ You can use the [TechPass portal](http://portal.techpass.gov.sg) to do a self-se
 
 ## Audience
 
-- Public officers using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name\>@\<acronym for your agency\>.gov.sg*.
+- Public officers using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name\>@\<acronym for your agency\>.gov.sg*. For example, peter_wilson@tech.gov.sg.
+
+- Vendors or contractors using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name>_FROM\>.\<vendor organisation name\>@\<acronym for the agency\>.gov.sg*. For example, peter_wilson_FROM.VENDORPROVIDER@tech.gov.sg.
```

Comment:
```suggestion
- Vendors, contractors or interns using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name>_FROM\>.\<vendor organisation name\>@\<acronym for the agency\>.gov.sg*. For example, peter_wilson_FROM.VENDORPROVIDER@tech.gov.sg.
```

Reply:
Are interns the Temp hires? i.e are you clubbing points 2 and 3? If that is the case the sample email address formats to be updated in point 2.